### PR TITLE
Feature/bc mobile support flow

### PR DESCRIPTION
**Pull request by nicktaras**

### CODE REVIEW

**Comment by nicktaras** [comment]
@snowwhitedev, @micwallace, @BankiLouis please see this PR to enable the Brand Connector demo to be able to mint tokens via desktop or mobile wallets.

File: ecommerce-store-website/src/base/utils/network.js [hunk 1]
```diff
@@ -5,7 +5,14 @@ export const chainMap = {
 	'0x3': 'Ropsten',
 	'0x5': 'Goerli',
 	'0x2a': 'Kovan',
-	'0x137': 'Polygon'
+	'0x137': 'Polygon',
```
[the remark this answers is not in the document]
Comment (nicktaras) [reply ecommerce-store-website/src/base/utils/network.js]: Good spot @snowwhitedev.

Screen Shot 2022-11-04 at 12 29 08 pm

[the remark this answers is not in the document]
Comment (snowwhitedev) [reply ecommerce-store-website/src/base/utils/network.js]: Looks good. Thank you

**Review by snowwhitedev — APPROVED** [review]
Looks good

**Pull request merged**